commit 75faaec96ec583f641b85cec23c904bb6ea0f4e7
Author:     Yuhong Wen <yuhongw@nvidia.com>
AuthorDate: Tue Apr 16 13:48:37 2024 -0400
Commit:     GitHub <noreply@github.com>
CommitDate: Wed Apr 17 01:48:37 2024 +0800

    Added more logging for the job status changing. (#2480)
    
    * Added more logging for the job status changing.
    
    * Fixed a logging call error.
    
    ---------
    
    Co-authored-by: Chester Chen <512707+chesterxgchen@users.noreply.github.com>
    Co-authored-by: Yuan-Ting Hsieh (謝沅廷) <yuantingh@nvidia.com>
---
 nvflare/private/fed/server/job_runner.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/nvflare/private/fed/server/job_runner.py b/nvflare/private/fed/server/job_runner.py
index 15a9bea8..d6bf91e8 100644
--- a/nvflare/private/fed/server/job_runner.py
+++ b/nvflare/private/fed/server/job_runner.py
@@ -444,6 +444,7 @@ class JobRunner(FLComponent):
                                     },
                                     fl_ctx,
                                 )
+                                self.log_info(fl_ctx, f"Updated the schedule history of Job: {job_id}")
 
                             if failed_clients:
                                 deployable_clients = {k: v for k, v in client_sites.items() if k not in failed_clients}
@@ -465,6 +466,7 @@ class JobRunner(FLComponent):
                             with self.lock:
                                 self.running_jobs[job_id] = ready_job
                             job_manager.set_status(ready_job.job_id, RunStatus.RUNNING, fl_ctx)
+                            self.log_info(fl_ctx, f"Job: {job_id} started to run, status changed to RUNNING.")
                         except Exception as e:
                             if job_id:
                                 if job_id in self.running_jobs:
